recipe/meta.yaml

```diff
@@ -12,41 +12,26 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 1001
   skip: true  # [win]
 
-requirements:
-  build:
-    - pkg-config
-    - make
-    - {{ compiler('c') }}
-    - bc
-  host:
-    - blas 1.1 openblas
-    - openblas
-    - gsl
-    - zlib
-    - fftw
-    - hdf5
-    - swig >=3.0.7
-
 outputs:
   - name: lal
     script: install-c.sh
     requirements:
-      host:
+      build:
         - {{ compiler('c') }}
         - make
+        - pkg-config
+      host:
         - swig >=3.0.7
-        - blas 1.1 openblas
-        - openblas
+        - mkl-devel  {{ mkl }}  # [blas_impl == "mkl"]
+        - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
```

Comment:
@scopatz, can you explain this change you proposed that introduces a dependency on `openblas-devel`? Is this just a way to get conda-forge to recognise that this package depends on `openblas`, but with everything setup so that we could easily build against `mkl` in the future?

Reply:
(I have proposed this change separately in conda-forge/lal-feedstock#14, and if that is merged cleanly, I will bot-rerun this PR).